osf/migrations/0215_auto_20200901_1344.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-09-01 13:44
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import django_extensions.db.fields
+import osf.models.base
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('osf', '0214_auto_20200701_1658'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='RegistrationRequestAction',
```

Comment:
Drive-by: I thought we didn't need the RequestActions. Is this going to change what we told front end, and is it going to affect the 'Integrate state machine' PR?

Reply:
ug good catch this was should be disappeared because it's just duplicated because the auto-generated names were different. So when I pulled from develop this didn't go away. But It should be fixed now.